css_tasks/task5/main.css

```diff
@@ -0,0 +1,283 @@
+  text-indent: 25px;
+  margin-top: 1.5rem;
+}
+.article__text::first-letter {
+  color: var(--mainThemeColor);
+  font-size: 3rem;
+  font-family: var(--headerFont);
+  font-weight: 700;
+}
+.article__blockquote {
+  font-style: italic;
+  font-size: 1.5rem;
+  line-height: 1.8rem;
+  color: var(--mainThemeColor);
+}
+/* SECONDARY SECTION */
+.content__secondary-section {
+  font-family: var(--mainThemeFont);
+  margin-top: 1.5rem;
+  display: flex;
+}
+.content__article--secondary {
+  width: 69%;
+  column-width: 250px;
+  column-gap: 20px;
+  column-rule: 1px solid var(--mainThemeColor);
+  margin-right: 2rem;
+}
+.article__header--secondary {
+  text-align: center;
+  font-family: var(--articleHeaderFont);
+  font-size: 1.5rem;
+  line-height: 1.6;
+  margin-top: 0;
+}
+.article__blockquote--secondary {
+  column-span: all;
+}
+.content__secondary--section__aside {
+  width: 30%;
+}
+.aside__image-container {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+}
+.article__image--secondary {
+  max-width: 100%;
+  max-height: 100%;
+}
+/* AD */
+.content__ad {
+  margin: 2rem 0 2rem 0;
+  width: 100%;
+  height: 500px;
+  background-image: url("carlsberg-min.jpg");
+  background-repeat: none;
+  background-position: center top;
+  background-size: cover;
+}
+/**********************
+*                     *
+*   INTERACTIONS      *
+*                     *
+***********************/
+
+/*  CHANGE THEME */
+.theme-toggler:checked ~ .header .header__top-bar {
+  background: var(--secondaryThemeColor);
+}
+.theme-toggler:checked ~ .header .header__info {
+  border-color: var(--secondaryThemeColor);
+}
+.theme-toggler:checked ~ .content .content__article {
+  column-rule: 2px solid var(--secondaryThemeColor);
+}
+.theme-toggler:checked ~ .content .content__article .article__blockquote {
+  color: var(--secondaryThemeColor);
+}
+.theme-toggler:checked
+  ~ .content
+  .content__article
+  .article__text::first-letter {
+  color: var(--secondaryThemeColor);
+}
+/* CHANGE FONT */
+.font-toggler:checked ~ .content .content__article {
+  font-family: var(--secondaryThemeFont);
+}
+.font-toggler:checked
+  ~ .content
+  .content__article
+  .article__text::first-letter {
+  font-family: var(--secondaryThemeFont);
+}
+/* CHANGE FONT-SIZE  */
+.font-size-toggler:checked ~ .content .content__article {
+  font-size: 1.2rem;
+}
+/**********************
+*                     *
+*   MEDIA QUERIES     *
+*                     *
+***********************/
+
+/* really basic stuff  */
+
+@media screen and (min-width: 1600px) {
+  .article__image {
+    max-width: 420px;
+  }
+}
+@media screen and (max-width: 1440px) {
+  .content__ad {
+    height: 400px;
+  }
+}
+/**********************
+*                     *
+*   MEDIA QUERIES     *
+*                     *
+***********************/
+
+/* Firefox fix for main article header */
+@supports not (column-span:all){
+  .content__article{
+    margin-top:10rem;
+  }
+  .article__header{
+    position:absolute;
```

Comment:
I understand that it is fix, but I think that there is other way to do it without absolute positioning, as it looks very strange

Reply:
probably yes, but I think I would have to change HTML structure.